posts/ja/2022-11-01-letstry-instant-on-intel-mac-en.adoc

```diff
@@ -0,0 +1,280 @@
+WARNING: Unknown module: jdk.attach specified to --add-exports
+Launching defaultServer (Open Liberty 22.0.0.11-beta/wlp-1.0.69.cl221020220912-1100) on Eclipse OpenJ9 VM, version 17.0.5-ea+2 (en_US)
+CWWKE0953W: This version of Open Liberty is an unsupported early release version.
+[AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/checkpoint.xml
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
+[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/ibm/api/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/health/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/metrics/
+[AUDIT   ] CWWKT0016I: Web application available (default_host): http://940fd476eccc:9080/dev/
+[AUDIT   ] CWWKZ0001I: Application guide-getting-started started in 1.340 seconds.
+[AUDIT   ] CWWKC0451I: A server checkpoint was requested. When the checkpoint completes, the server stops.
+/opt/ol/wlp/bin/server: line 946:   130 Killed                  "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null 
+----
+
+Since the container was started without `--rm`` this time, the container environment after execution remains as it is. In this environment, files that record process information with checkpoint are saved.
+
+[source]
+----
+$ podman ps -a
+CONTAINER ID  IMAGE                             COMMAND               CREATED        STATUS                    PORTS       NAMES
+940fd476eccc  localhost/getting-started:latest  /opt/ol/wlp/bin/s...  4 minutes ago  Exited (0) 4 minutes ago              getting-started-checkpoint-container
+----
+
+The following command commits and register it as a new image.
+
+[source]
+----
+$ podman commit getting-started-checkpoint-container getting-started-instanton
+a856d767b8c31718dfbc6e60f742675448086fb4421490b5bfde6d3392d2f879
+$ podman images
+REPOSITORY                           TAG             IMAGE ID      CREATED         SIZE
+localhost/getting-started-instanton  latest          a856d767b8c3  7 seconds ago   990 MB
+localhost/getting-started            latest          1049db82664e  31 minutes ago  890 MB
+icr.io/appcafe/open-liberty          beta-instanton  7cd2ddfe28da  2 weeks ago     864 MB
+----
+
+== Starting Open Liberty using the InstantOn feature 
+
+Now it's time to execute InstantOn. When you start Liberty using this image, the server will start in a flash.
+
+[source]
+----
+% podman run -it --rm --privileged -p 9080:9080 getting-started-instanton
```

Comment:
Have you tried this without `--privileged`.  What about:
```
podman run \
  --rm \
  --cap-add=CHECKPOINT_RESTORE \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_PTRACE \
  -p 9080:9080 \
  getting-started-instanton
```

Reply:
Yes I had the same question as Tom, it would be good to talk about the "unprivileged restore" being mentioned here in some way (either as something you tried and it worked, OR as something that is possible on Linux (as discussed in the original Liberty InstantOn blog) but needs further investigation on Mac).

Reply:
Currently trying to chat with the techsales team. leaving the conversation open.

Reply:
I was able to try and restore worked. The content was updated with more restore without --privileged info. Spell corrected